applications/nrf5340_audio/src/audio/sw_codec_select.c

```diff
@@ -9,9 +9,12 @@
 #include <zephyr/kernel.h>
 #include <errno.h>
 #include <pcm_stream_channel_modifier.h>
+#include <audio_module/audio_module.h>
```

Comment:
No the audio module is below this.
However if we have an audio framework then sw_codec_select would be below it.
Remember this is not a framework, but a module/component interface. A frame work would allow these modules to be connected together, send data along, control modules, etc.